test/jdk/java/awt/print/Dialog/DialogType.java

```diff
@@ -26,49 +26,61 @@
  * @bug 6568874
  * @key printer
  * @summary Verify the native dialog works with attribute sets.
- * @run main/manual=yesno DialogType
+ * @library /java/awt/regtesthelpers /test/lib
+ * @build PassFailJFrame
+ * @run main/manual DialogType
  */
 
-import java.awt.print.*;
-import javax.print.attribute.*;
-import javax.print.attribute.standard.*;
+import java.awt.print.PrinterJob;
 
-public class DialogType {
+import javax.print.attribute.Attribute;
+import javax.print.attribute.HashPrintRequestAttributeSet;
+import javax.print.attribute.PrintRequestAttributeSet;
+import javax.print.attribute.standard.DialogTypeSelection;
 
-  static String[] instructions = {
-     "This test assumes and requires that you have a printer installed",
-     "It verifies that the dialogs behave properly when using new API",
-     "to optionally select a native dialog where one is present.",
-     "Two dialogs are shown in succession.",
-     "The test passes as long as no exceptions are thrown, *AND*",
-     "if running on Windows only, the first dialog is a native windows",
-     "control which differs in appearance from the second dialog",
-     ""
-  };
+import jtreg.SkippedException;
 
-  public static void main(String[] args) {
+public class DialogType {
+    private static PrinterJob job;
 
-      for (int i=0;i<instructions.length;i++) {
-         System.out.println(instructions[i]);
-      }
+    private static final String INSTRUCTIONS = """
+        This test assumes and requires that you have a printer installed
+        It verifies that the dialogs behave properly when using new API
+        to optionally select a native dialog where one is present.
+        Two dialogs are shown in succession.
+        The test passes as long as no exceptions are thrown, *AND*
+        if running on Windows only, the first dialog is a native windows
+        control which differs in appearance from the second dialog
```

Comment:
```suggestion
        The test passes as long as no exceptions are thrown.
```
Platform specific instructions can be added as a NOTE. It would be better if it is mentioned at the start of instruction.

The first and second dialog differs on MacOS as well. Not checked on Linux.

Current instruction says only about Windows, better to verify and modify the instruction.

Reply:
Reconfirmed by testing on Mac OS and Linux as well - both of these have different dialogs for first and second.
As suggested, added a note. But I have added it at the end as it seemed to be better to first step through the instructions and then give an OS specific context for those instructions.